docs/reference/rest-api/security/update-settings.asciidoc

```diff
@@ -18,27 +18,28 @@ Updates the settings of the security internal indices.
 ==== {api-request-body-title}
 
 `security`::
-(Optional, object) Settings to be used for the index used for most security 
+(Optional, object) Settings to be used for the index used for most security
 configuration, including Native realm users and roles configured via the API.
 
 `security-tokens`::
-(Optional, object) Settings to be used for the index used to store 
+(Optional, object) Settings to be used for the index used to store
 <<security-api-get-token,tokens>>.
 
-`security`::
-(Optional, object) Settings to be used for the index used to store 
+`security-profile`::
```

Comment:
This is change 1.

Reply:
Truly, the most pernicious class of error: the copy/paste error.

🤦🏼‍♀️ thanks